<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tfs::Internal::OptionsPage</class>
 <widget class="QWidget" name="Tfs::Internal::OptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="configGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>17</y>
     <width>599</width>
     <height>57</height>
    </rect>
   </property>
   <property name="title">
    <string>Configuration</string>
   </property>
   <layout class="QFormLayout" name="formLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="tfsCommandLabel">
      <property name="text">
       <string>Command:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="Utils::PathChooser" name="commandChooser" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="userGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>599</width>
     <height>108</height>
    </rect>
   </property>
   <property name="title">
    <string>User</string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="defaultUsernameLabel">
      <property name="toolTip">
       <string>Username to use by default on commit.</string>
      </property>
      <property name="text">
       <string>Default username:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="defaultUsernameLineEdit">
      <property name="toolTip">
       <string>Username to use by default on commit.</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="defaultEmailLabel">
      <property name="toolTip">
       <string>Email to use by default on commit.</string>
      </property>
      <property name="text">
       <string>Default email:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="defaultEmailLineEdit">
      <property name="toolTip">
       <string>Email to use by default on commit.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
   <slots>
    <signal>editingFinished()</signal>
    <signal>browsingFinished()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
